src/nginx/request.cc

```diff
@@ -33,6 +33,65 @@ namespace google {
 namespace api_manager {
 namespace nginx {
 
+namespace {
+
+ngx_int_t ngx_list_delete_elt(ngx_list_t *list, ngx_list_part_t *cur,
```

Comment:
Did you copy the code from Nignx?  If so, please add comment on where the code is copied from.

Reply:
It is not from the ngix. ngx_list doesn't support remove function.

Reply:
Removed new function ngx_list_delete*